From d6f75254e35d22fadf7b79567958942cb1fe0aed Mon Sep 17 00:00:00 2001
From: msx98 <git@lx.ax>
Date: Wed, 11 Feb 2026 14:08:19 +0000
Subject: [PATCH 1/1] revive emmc patch

---
 drivers/mmc/core/block.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/mmc/core/block.c b/drivers/mmc/core/block.c
index 3564a0f63..9ef5de93f 100644
--- a/drivers/mmc/core/block.c
+++ b/drivers/mmc/core/block.c
@@ -1028,13 +1028,14 @@ static unsigned int mmc_blk_data_timeout_ms(struct mmc_host *host,
 static int mmc_blk_reset(struct mmc_blk_data *md, struct mmc_host *host,
 			 int type)
 {
-	int err;
+	int err, i;
 	struct mmc_blk_data *main_md = dev_get_drvdata(&host->card->dev);
 
 	if (md->reset_done & type)
 		return -EEXIST;
 
 	md->reset_done |= type;
+	for (i=0, err=1; (err!=0) && (i<2); i++)
 	err = mmc_hw_reset(host->card);
 	/*
 	 * A successful reset will leave the card in the main partition, but
-- 
2.47.3

