From c5e8773074c94b1fd120d36e786d1c3032674bab Mon Sep 17 00:00:00 2001
From: msx98 <git@lx.ax>
Date: Wed, 11 Feb 2026 14:15:18 +0000
Subject: [PATCH 1/1] dont flush cache on reset or something

---
 drivers/mmc/core/mmc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mmc/core/mmc.c b/drivers/mmc/core/mmc.c
index 7e39017e4..ff341d135 100644
--- a/drivers/mmc/core/mmc.c
+++ b/drivers/mmc/core/mmc.c
@@ -2256,7 +2256,7 @@ static int _mmc_hw_reset(struct mmc_host *host)
 	 * In the case of recovery, we can't expect flushing the cache to work
 	 * always, but we have a go and ignore errors.
 	 */
-	_mmc_flush_cache(host);
+	//_mmc_flush_cache(host);
 
 	if ((host->caps & MMC_CAP_HW_RESET) && host->ops->card_hw_reset &&
 	     mmc_can_reset(card)) {
-- 
2.47.3

